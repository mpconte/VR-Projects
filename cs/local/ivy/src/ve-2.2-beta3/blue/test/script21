#
# Testing weird problem with while conditions...
#
set l {a b c d e f g h i j k l m n o p q r s t u v w x y z}
set z [llength $l]
set i 0
while { $i < $z } {
    [stdout] writeln "i = $i, z = $z"
    [stdout] writeln "l{$i} = [lindex $l $i]"
    set i [expr {$i + 2}]
    [stdout] writeln [expr {$i < $z}]
}
