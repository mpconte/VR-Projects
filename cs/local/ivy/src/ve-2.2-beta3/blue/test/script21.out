i = 0, z = 26
l{0} = a
1
i = 2, z = 26
l{2} = c
1
i = 4, z = 26
l{4} = e
1
i = 6, z = 26
l{6} = g
1
i = 8, z = 26
l{8} = i
1
i = 10, z = 26
l{10} = k
1
i = 12, z = 26
l{12} = m
1
i = 14, z = 26
l{14} = o
1
i = 16, z = 26
l{16} = q
1
i = 18, z = 26
l{18} = s
1
i = 20, z = 26
l{20} = u
1
i = 22, z = 26
l{22} = w
1
i = 24, z = 26
l{24} = y
0
result = ok
